README.md

```diff
@@ -31,10 +31,14 @@ Use the `all-languages` feature to enable all languages.
 
 This crate supports Rust v1.41.1 and up and works with `no_std`.
 
-When using older version of Rust, you might have to pin the version of the
-`bitcoin_hashes` crate used as such:
-```
-$ cargo update --package "bitcoin_hashes" --precise "0.12.0"
+When using older version of Rust, you might have to pin the versions of several crates, for an up-to-date list refer to [`contrib/test.sh`](contrib/test.sh):
+
+```bash
+cargo update --package "bitcoin_hashes" --precise "0.12.0"
```

Comment:
Why not use and put `Cargo.lock` under revision?

Reply:
If some downstream project needs pinning, they probably will not even know that they should look for pinning instruction in some dependency between what failed and their own code (and which one exactly). And figuring it out manually is not a rocket science.

I don't know where this idea that bip39 is somehow MSRV coordinator of rust-bitcoin ecosystem's MSRV. If anything it's own responsibility should be just ensuring it compiles at the minimum desired MSRV, and that's about it. `Cargo.lock` would achieve that.

Reply:
I don't mind too much either way, but if you have a weird MSRV you probably have to add every dependency very carefully anyway and the deps telling you which of their transitive dependencies are problematic is useful. I was tinkering around for quite a bit till I got it to work.

Note that even before at least `bitcoin_hashes` was already being pinned like that, so I'm not introducing this as a new concept but rather extending it to the crates that became problematic since the test last worked. 

Maybe @apoelstra or @tcharding have an opinion on this one.

Reply:
Agreed. It's true that downstream projects will have a hard time finding instructions in the `README`s of their deps, but they'll have an even harder time finding anything in the `Cargo.lock` files of their deps.

It's probably useful to commit a `Cargo-minimal.lock` file here for use in CI the way we do in rust-bitcoin, but IMO committing a file called `Cargo.lock` would cause more confusion than help.

Reply:
Yeah, that's fair. So I'd be fine if there were a committed `Cargo.lock` file, and I suppose it ought to be one that works with the project's MSRV.

Reply:
I added a `Cargo-minimal.lock` and a test that it's still correct for MSRV.